src/extensions/extensionsHelper.ts

```diff
@@ -532,7 +532,8 @@ export async function publishExtensionVersionFromLocalSource(args: {
         `${args.publisherId}/${args.extensionId}`
       )}'. Please make sure this version is greater than the current version (${clc.bold(
         extension.latestVersion
-      )}) inside of extension.yaml.\n`
+      )}) inside of extension.yaml.\n`,
+      { exit: 104 }
```

Comment:
How was 104 picked? Should we move this to a constant somewhere?

Reply:
103 is used below... 104 is unused. Single use constants are usually frowned upon.